src/views/EnrollmentCheckIn/components/CompletedCheckIn/CompletedCheckIn.module.scss

```diff
@@ -1,3 +1,13 @@
+.button_container {
+  Button:disabled {
```

Comment:
Is this being used somewhere, it doesn't look like correct CSS syntax here, so I wanted to make sure it was working how you intended...

Reply:
There's a call for it in the file, I just changed this file to fit more in inline with other css files and changed the call accordingly.